Evergreen.xcodeproj/project.pbxproj

```diff
@@ -813,19 +850,37 @@
 			productReference = 849C64711ED37A5D003D8FC0 /* EvergreenTests.xctest */;
 			productType = "com.apple.product-type.bundle.unit-test";
 		};
+		9451C1EA1EE076F200605914 /* EvergreenUITests */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 9451C1F21EE076F200605914 /* Build configuration list for PBXNativeTarget "EvergreenUITests" */;
+			buildPhases = (
+				9451C1E71EE076F200605914 /* Sources */,
+				9451C1E81EE076F200605914 /* Frameworks */,
+				9451C1E91EE076F200605914 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				9451C1F11EE076F200605914 /* PBXTargetDependency */,
+			);
+			name = EvergreenUITests;
+			productName = EvergreenUITests;
+			productReference = 9451C1EB1EE076F200605914 /* EvergreenUITests.xctest */;
+			productType = "com.apple.product-type.bundle.ui-testing";
+		};
 /* End PBXNativeTarget section */
 
 /* Begin PBXProject section */
 		849C64581ED37A5D003D8FC0 /* Project object */ = {
 			isa = PBXProject;
 			attributes = {
-				LastSwiftUpdateCheck = 0820;
+				LastSwiftUpdateCheck = 0830;
 				LastUpgradeCheck = 0820;
 				ORGANIZATIONNAME = "Ranchero Software";
 				TargetAttributes = {
 					849C645F1ED37A5D003D8FC0 = {
 						CreatedOnToolsVersion = 8.2.1;
-						DevelopmentTeam = M8L2WTLA8W;
+						DevelopmentTeam = PZ4HKYD5WL;
```

Comment:
Changes like this might break @brentsimmons branch since he's likely on a different dev team than you are.